src/sqlancer/stonedb/gen/StoneDBTableCreateGenerator.java

```diff
@@ -222,8 +224,7 @@ private void appendColumnOption(StoneDBDataType type) {
                 }
                 break;
             case COMMENT:
-                // TODO: generate randomly
-                sb.append(String.format("COMMENT '%s' ", "asdf"));
+                sb.append(String.format("COMMENT '%s' ", new Randomly().getString()));
```

Comment:
Rather than creating new `Randomly` here, use the one from the global state object.